hicexplorer/hicFindTADs.py

```diff
@@ -17,6 +17,7 @@ def parse_arguments(args=None):
     parser = argparse.ArgumentParser(
         formatter_class=argparse.RawDescriptionHelpFormatter,
         description="""
+branch : new
```

Comment:
@bgruening sorry that was my quick test after installing that I am running the right script. (due to so many python envs on my system). Removed that now